Error on build output

Building the code with Maven (`mvn package`) on a Windows machine gives the following error (the build itself succeeds):

```
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative-reference (starting with slash) /site
[INFO] Building zip: C:\Users\B004114\Documents\GitHub\heidivanparys\ets-ogcapi-features10\target\ets-ogcapi-features10-1.1-SNAPSHOT-ctl.zip
[INFO] Building jar: C:\Users\B004114\Documents\GitHub\heidivanparys\ets-ogcapi-features10\target\ets-ogcapi-features10-1.1-SNAPSHOT-aio.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```

--- a/src/assembly/ctl-scripts.xml
+++ b/src/assembly/ctl-scripts.xml
@@ -9,11 +9,15 @@
   </formats>
   <includeBaseDirectory>true</includeBaseDirectory>
   <baseDirectory>${ets-code}/${spec-version}</baseDirectory>
-  <includeSiteDirectory>true</includeSiteDirectory>
   <fileSets>
     <fileSet>
       <directory>${project.build.scriptSourceDirectory}/ctl</directory>
       <outputDirectory>ctl</outputDirectory>
+      <filtered>true</filtered>
+    </fileSet>
+    <fileSet>
+      <directory>${project.build.directory}/site</directory>
+      <outputDirectory>site</outputDirectory>
       <filtered>true</filtered>
     </fileSet>
   </fileSets>

